### ISSUE TITLE
Group Properties Should Match Create Parameters

### ISSUE THREAD

**Issue by BrettFrahm** [Esri/arcgis-python-api#1898]
**Is your feature request related to a problem? Please describe.**
Groups are becoming "corrupted" according to ESRI Support and stop syncing over features correctly with collaborations. I need to recreate groups with the same settings and shared items in the same portal. I am not transferring sites/portals so it is a simpler use case than what is needed with cloning a group or using the GroupMigrationManager. With that, I am using the GroupManager.create() to make a new group with the same thumbnail, settings, etc. as an existing group.

The problem is that there doesn't seem to be group properties for every parameter that is used to create a group.

```python
<old_group = arcgis.gis.Group(old_gis, old_id)
old_group_members = old_group.get_members()
        old_thumbnail = old_group.download_thumbnail()>
new_group = new_gis.groups.create(title=new_title, tags=old_group['tags'],
                                              description=old_group['description'],snippet=old_group['snippet'],
                                              access= old_group['access'], thumbnail=old_thumbnail,
                                              is_invitation_only=old_group['isInvitationOnly'], sort_field=old_group['sortField'],
                                              sort_order=old_group['sortOrder'], is_view_only=old_group['isViewOnly'],
                                              auto_join=old_group['autoJoin'], provider_group_name=old_group['providerGroupName'],
                                              provider=old_group['provider'])
```

**Describe the solution you'd like**
Add properties to Group object that match the following GroupManager.create() parameters.

- max_file_size
- display_settings
- is_open_data
- leaving_disallowed
- membership_access

**Comment by achapkowski** [comment]
What version of the python api are you using (example:2.3.1)?  
What version of enterprise?

**Comment by BrettFrahm** [comment]
Hello @achapkowski ,

We are on 2,2,0,1 at the moment. Our Enterprise is at 11.2

**Comment by nanaeaubry** [comment]
Please log this with support. We are currently developing for the latest Enterprise versions.

**Comment by BrettFrahm** [comment]
@nanaeaubry Thank you for responding to this older enhancement, but I am a little confused. 

- This enhancement should be relevant to ArcGIS Online or latest Enterprise version if it has not been implemented yet. My request is to add properties to Group object that match the GroupManager.create() parameters.

- I've been told multiple times that this Github is the best place to log known bugs or request enhancements due to how active the devs are here. I am happy to get this logged with support as a Enhancement and I often do this.

**Comment by achapkowski** [comment]
@BrettFrahm 2.2.x is an older series and we won't be backporting the API to that version.  Can you try using 2.4.x version of the API?

**Comment by nanaeaubry** [comment]
@BrettFrahm Yes this is a great place for quick issues and questions for us to help with. 
However, for more in depth enhancement requests we do encourage support since they have a greater capacity to investigate the enhancement, gather information on who else would like it, and assign a priority to it.

The `Group` class inherits from dictionary so you can try to do: 
```
grp = group_manager.get(<grp_id>)
dict(grp)
```

This returns the group properties as a dictionary. We have a method in the `GroupManager` called: `create_from_dict` which could be a possible solution for your use case as well.

Hope that helps a bit
